<!-- This file shows the most important parameters in their default settings,
     to make them easily available for beginners.

     The openni driver is started automatically -->
<launch>
  <include file="$(find astra_launch)/launch/astra.launch"/>
  <node pkg="ORB_SLAM2" type="RGBD" name="ORB_SLAM2" args="$(find ORB_SLAM2)/Data/ORBvoc.bin $(find ORB_SLAM2)/Data/Settings.yaml" output="screen" > 
  <remap from="/camera/rgb/image_raw" to="camera/rgb/image_rect_color"/>
  <remap from="camera/depth_registered/image_raw" to="camera/depth_registered/sw_registered/image_rect_raw"/>
  <param name="use_pango_viewer" type="bool" value="false" />
  <param name="use_imu_data" type="bool" value="false" />
  </node>
</launch>
